app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -123,12 +125,16 @@ $white       : #ffffff;
 // $warning-color: #f08a24;
 // $info-color: #a0d3e8;
 
-$primary-color: #ffa133;
+$primary-color: #edb97d;
+$secondary-color: #769481;
+
 
 // We use these to control various global styles
 // $body-bg: #008CBA;
+$body-bg: #f5e5c4;
 // $body-font-color: $jet;
 // $body-font-family: $font-family-sans-serif;
+$body-font-family: Copperplate,Copperplate Gothic Light,fantasy;;
```

Comment:
Commas in lists should be followed by a single space<br>Declaration should be terminated by a single semicolon